boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7_defconfig

```diff
@@ -20,3 +20,5 @@ CONFIG_UART_CONSOLE=y
 
 # Enable GPIO
 CONFIG_GPIO=y
+
+CONFIG_ETH_STM32_HAL_PHY_ADDRESS=1
```

Comment:
Generates following error:
```
warning: ETH_STM32_HAL_PHY_ADDRESS (defined at drivers/ethernet/Kconfig.stm32_hal:67) was assigned
the value '1' but got the value ''. Check these unsatisfied dependencies: ETH_STM32_HAL (=n),
ETH_DRIVER (=n). See
```

To be put in https://github.com/zephyrproject-rtos/zephyr/blob/75e0c29baa642b053deffa7127f3ab8da23fb0d6/boards/st/stm32h745i_disco/Kconfig.defconfig#L9-L17

Also, it would be nice to add a comment t mentionned this is linked with address defined in dt

Reply:
It should be fixed now. I also rebased to current main